pyccl/halos/halo_model.py

```diff
@@ -995,7 +995,9 @@ def halomod_Tk3D_SSC(cosmo, hmc,
                      normprof3=False, normprof4=False,
                      p_of_k_a=None, lk_arr=None, a_arr=None,
                      extrap_order_lok=1, extrap_order_hik=1,
-                     use_log=False):
+                     use_log=False, is_clustering1=False,
```

Comment:
Can we call this `is_number_counts` or `is_nc`? Just to keep it in agreement with the nomenclature we use in tracers.py

Reply:
Done